guides/common/modules/proc_uploading-content-to-deb-repositories-by-using-cli.adoc

```diff
@@ -0,0 +1,26 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="uploading-content-to-deb-repositories-by-using-cli"]
+= Uploading content to Deb repositories by using Hammer CLI
+
+[role="_abstract"]
+You can upload individual Deb packages to Deb repositories by using Hammer CLI.
```

Comment:
Optional (because we're not there with the other abstracts yet and it wouldn't be fair to require this at the moment): Ideally, an abstract should also clarify the use case: when and why a user might want to follow the procedure, not just what the procedure does.

For example `when`: Is the "individual packages" part important here? Is this what distinguishes this procedure from other procedures in the guide? When would a user want to upload an individual package?

Or `why`: What will happen when the package will be uploaded? Can we explicitly specify that the package will then be available for hosts?